### DISCUSSION TITLE
Rouge PAN Broadcast Messages disables NLG Lamps - MAC no ack' (233) Error

### DISCUSSION THREAD

**Discussion by yonz2** [Koenkk/zigbee2mqtt#16257] [Devices]
I have been battling with my Paul Neuhaus lamps for quite some time. After joining one of them to my Zigbee Network (Home Assistant running zigbee2MQTT - latest DEV version), I am able to control it for about 5 minutes then the lamp simply "locks up". All attempts to send commands to it result in a "`'MAC no ack' (233) Error`". 
I got myself a sniffer from China and started to look at what was happening on the zigbee network. This is where things got "interesting".
Every five minutes or so, a PAN Broadcast packet (`Destination PAN = 0xffff`) is sent out from a PAN Network I don't have (`Source PAN = 0xe4c0`. My PAN is `0x80d6`. 

After this broadcast message is sent out the newly joined NLG Lamps stop working and return `'MAC no ack' (233) Error`

**So my question is, where is this coming from?**
**What is the rogue PAN `0xe4c0`?**

What can I do about it? ( I tried to blacklist the Source IEEE Address in zigbee2MQTT, but that didn't have any impact)

Any help/hints will be much appreciated,

Thanks in advance
Yonz

Configuration of one of the three NLG TW Lamps (all three exhibit exactly the same behaviour):
```
Friendly name: Esstisch
Description: 
Last seen: N/A
Availability: Disabled
Device type: Router
Zigbee Model: NLG-TW light
Zigbee Manufacturer: Neuhaus Lighting Group
Description: Various tunable white lights (e.g. 8195-55)
Support status: Supported

IEEE Address: 0x90940300e00bfdb0
Network address: 0x51BB
Firmware build date: 20210611
Firmware version: 1.0.6
Manufacturer: Paul Neuhaus
Model: NLG-TW light
Power: Mains
Interview completed: True
```

BTW: 
1. I scanned all Zigbee Networks from Channels 11 to 25 but none of them hat any traffic. (except mine, of course)
2. I tried different Channels (11 and 20) and made sure there is no interference with my WiFi (2.4GHz WiFi is on Channel 1; Zigbee on Channel 20)
3. I tried two different Zigbee Controller, Sonoff Dongle 3.0 as well as Sonoff Zigbee Bridge (flashed with Tasmota)

(Here's a PCAP file with some of those messages: [rouge.pcapng.gz](https://github.com/Koenkk/zigbee2mqtt/files/10452193/rouge.pcapng.gz)

Here's a JSON Dump of one of those messages:
```
[
  {
    "_index": "packets-1970-01-01",
    "_type": "doc",
    "_score": null,
    "_source": {
      "layers": {
        "frame": {
          "frame.interface_id": "0",
          "frame.interface_id_tree": {
            "frame.interface_name": "-"
          },
          "frame.encap_type": "104",
          "frame.time": "Jan  1, 1970 01:38:22.161287000 CET",
          "frame.offset_shift": "0.000000000",
          "frame.time_epoch": "2302.161287000",
          "frame.time_delta": "0.000000000",
          "frame.time_delta_displayed": "0.000000000",
          "frame.time_relative": "0.000000000",
          "frame.number": "1",
          "frame.len": "89",
          "frame.cap_len": "89",
          "frame.marked": "0",
          "frame.ignored": "0",
          "frame.protocols": "wpan:data"
        },
        "wpan-zboss": {
          "wpan-zboss.direction": "0x00",
          "wpan-zboss.page": "0",
          "wpan-zboss.channel": "20",
          "wpan-zboss.trace": "0"
        },
        "wpan": {
          "wpan.frame_length": "76",
          "wpan.fcf": "0xd809",
          "wpan.fcf_tree": {
            "wpan.frame_type": "0x0001",
            "wpan.security": "1",
            "wpan.pending": "0",
            "wpan.ack_request": "0",
            "wpan.pan_id_compression": "0",
            "wpan.fcf.reserved": "0",
            "wpan.seqno_suppression": "0",
            "wpan.ie_present": "0",
            "wpan.dst_addr_mode": "0x0002",
            "wpan.version": "1",
            "wpan.src_addr_mode": "0x0003"
          },
          "wpan.seq_no": "125",
          "wpan.dst_pan": "0xffff",
          "wpan.dst16": "0xffff",
          "wpan.addr16": "0xffff",
          "wpan.src_pan": "0xe4c0",
          "wpan.src64": "9e:da:27:d2:ce:5c:45:3e",
          "wpan.addr64": "9e:da:27:d2:ce:5c:45:3e",
          "wpan.aux_sec.hdr": {
            "wpan.aux_sec.security_control_field": "0x15",
            "wpan.aux_sec.security_control_field_tree": {
              "wpan.aux_sec.sec_level": "0x05",
              "wpan.aux_sec.key_id_mode": "0x02",
              "wpan.aux_sec.frame_counter_suppression": "0",
              "wpan.aux_sec.asn_in_nonce": "0",
              "wpan.aux_sec.reserved": "0x00"
            },
            "wpan.aux_sec.frame_counter": "52556",
            "Key Identifier Field": {
              "wpan.aux_sec.key_source": "0x00000000ffffffff",
              "wpan.aux_sec.key_source.bytes": "ff:ff:ff:ff",
              "wpan.aux_sec.key_index": "0xff"
            }
          },
          "wpan.mic": "5b:01:e2:b7",
          "_ws.expert": {
            "wpan.decrypt_error": "",
            "_ws.expert.message": "No encryption key set - can't decrypt",
            "_ws.expert.severity": "6291456",
            "_ws.expert.group": "83886080"
          },
          "TI CC24xx-format metadata: FCS OK": {
            "wpan.fcs_ok": "1",
            "wpan.rssi": "-24",
            "wpan.correlation": "107"
          }
        },
        "data": {
          "data.data": "23:5d:e8:73:f4:63:18:40:b6:37:32:42:a6:59:56:af:a8:b5:70:9a:23:7d:0f:44:ed:d4:13:8a:25:14:aa:34:29:06:24:5a:88:86:c1:d2:73:a5:48:ad:13",
          "data.len": "45"
        }
      }
    }
  }
]
```

**Comment by yonz2** [comment]
Update:
Turned out that this is a coexistence problem btw. NLG Lamps and Apple Homepod Mini. The Homepod Mini broadcasts a message to all PANs (Destination PAN: 0xFFFF) ca. every 5 minutes. All other ZIGBEE devices ignore this, except the NLG Lamps, they simply freeze. The only way around this is then to unplug the Apple Homepod Mini, and re-join the NLG lamps to the Zigbee Network. (It is then a question of Apple Homepod Mini or NLG Lamps, only one of them works at the same time)

BR
Yonz

**Comment by c-thiel** [comment]
I can reproduce the issue with my Homepod Mini as well.
What did Paul Neuhaus do there?

**Reply by yonz2** [reply]
I contacted NLG via E-Mail, Support Tickets etc. They more or less said there was no update available. However, I had seen in another thread that there was one! I then went back to the shop where I bought the Lamps, and had them contact NLG. After some discussions, NLG Agreed to replace the receivers. The new ones have version 1.2 (Build 1.2 from 17.04.2023). 
Now all NLG Lamps work when together with my HomePod Minis

![image](https://github.com/Koenkk/zigbee2mqtt/assets/13886257/bad50d94-0198-46db-b419-7f09918e0f69)

**Reply by c-thiel** [reply]
I have to say Paul Neuhaus support was really quick here and sent me some replacements.
It's working now like a breeze!
Thanks for the tip!

**Reply by Diaoul** [reply]
I'm having the same issue here.

@yonz2 did you only replace the receiver or the whole lamp? Any chance that you have a ticket number or something that I could reuse to make the same request?

Thanks

**Reply by yonz2** [reply]
@Diaoul Only the receiver... You need to get to at least a version 1.2

**Reply by Diaoul** [reply]
@yonz2 thanks for the info! Would you happen to still have the ticket number? I created one but I fear they will fail to understand what I'm saying :fearful: 
I have many Paul Neuhaus lamps, it's a shame they don't support OTA :facepalm:

**Comment by banray** [comment]
Also have the same issue. Tried Hue Bridge, Zigbee2Mqtt and ZHA. Connection drops between 1-4 minutes. After power cycle it works again for 1-4 minutes. Lamp is a Paul Neuhaus Q-Adriana LED floorlamp. Shows itself on as NLG-RGB-TW with firmware 1.0.4. Is there a way to update the firmware?

**Reply by yonz2** [reply]
Hi, 

No there is no way to update the Firmware, other than replacing the receiver. NLG was probably to lazy to implement the Full Zigbee support and skipped the OTA Update (even the cheap Chinese Lamps have it.... why not NLG??? And we are really paying a premium price for those things...)

**Reply by Diaoul** [reply]
Maybe you can hack into it if your savvy enough. I imagine this is mostly the same hardware as the newer versions, just a bump of software.

**Reply by banray** [reply]
Got my NLG lamps replaced today. Both of them have the newer Zigbee firmware. Everything works as it should now.

**Reply by pilot1297** [reply]
Hi there. I´ve also got the new Receiver but with firmware version 1.4.0 and got the same problem as before that the light become unresponsive. The only thing different this time is that it becomes after around 10 min unresponsive but is still some how connected to ZHA and can be revived with a SW-Build pull from the ZigBee-Manager. Anybody got the same Firmware or problem? 
I will also write the support again and keep you updated.
